requirements/pip.in

```diff
@@ -0,0 +1,110 @@
+# Base packages
+pip
+virtualenv
+
+django~=3.2.0
```

Comment:
Used to be `<3.3` but I guess this is the same.

Reply:
Yes. It's the same.